source/Calamari.Common/Features/StructuredVariables/StructuredConfigVariablesFileFormats.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Calamari.Common.Features.StructuredVariables
+{
+    public static class StructuredConfigVariablesFileFormats
+    {
+        public static readonly string Json = "Json";
```

Comment:
Implemented as `static readonly` as usual for Calamari to prevent inlining across assemblies.